README.md

```diff
@@ -3,12 +3,45 @@ NaiveBayes.jl
 
 [![Build Status](https://travis-ci.org/dfdx/NaiveBayes.jl.svg)](https://travis-ci.org/dfdx/NaiveBayes.jl)
 
-Naive Bayes classifier. Currently 3 types of NB are supported:
+Naive Bayes classifier. Currently 4 types of NB are supported:
 
  * **MultinomialNB** - Assumes variables have a multinomial distribution. Good for text classification. See `examples/nums.jl` for usage.
  * **GaussianNB** - Assumes variables have a multivariate normal distribution. Good for real-valued data. See `examples/iris.jl` for usage.
- * **KernelNB** - Computes kernel density estimates for each class. Good for data with continuous distributions.
+ * **KernelNB (depricated)** - Computes kernel density estimates for each class. Good for data with continuous distributions. 
+ * **HybridNB** - A hybrid empirical naive Bayes model for a mixuture of continues and discrete features. The continuous feaures are estimated using Kernel Density Estimation (simirarly to KernelBN). Note that the fit/predict methods take a vectors of feature vector `Vector{Vector}` rather than a matrix.
```

Comment:
mixuture -> mixture
continues -> continuous
feaures -> features
simirarly -> similarly (actually remove this part if we remove KernelNB)

Change -> Note: that the fit/predict methods take `Vector{Vector}` rather than a `Matrix`.

Also, it would be good to have something say that `Vector{Int}` are considered to be discrete and will compute a histogram rather than KDE.